src/main/webapp/WEB-INF/tags/instructor/results/sectionPanel.tag

```diff
@@ -26,7 +26,7 @@
                     </div>
                     <div class="col-sm-3">
                         <div class="pull-right">
-                            <a class="btn btn-success btn-xs" id="collapse-panels-button-section-${sectionIndex}" data-toggle="tooltip" title="Collapse or expand all ${isGroupedByTeam? 'team' : 'student'} panels. You can also click on the panel heading to toggle each one individually.">
+                            <a class="btn btn-success btn-xs" id="collapse-panels-button-section-${sectionIndex}" data-toggle="tooltip" title="Collapse or expand all ${isGroupedByTeam? 'team' : 'student'} panels. You can also click on the panel heading to toggle each one individually." style="display: none;">
```

Comment:
Use a class instead, I am quite sure we have a class which sets the CSS of ```display: none;```, the side effect of using a style like that is that when it is removed for any reason, we get the empty style which is not desirable